### ISSUE TITLE
atomicapp always tries to pull images - even if they are already local

### ISSUE THREAD

**Issue by dustymabe** [projectatomic/atomicapp#248]
In `nulecule_base.py` we check to see if an image is local using `docker images -q $name`. If an image is returned then no attempt is made to pull the image from a registry. Unfortunately unless the name of the image that we pass in is the full name (including registry) of the image then `docker images -q` returns nothing and we will always attempt to download the new image.

In the following case I already had the `projectatomic/mariadb-centos7-atomicapp` image downloaded:

```
[root@f22 rpm]$ docker images  | grep mariadb-centos7-atomicapp
docker.io/projectatomic/mariadb-centos7-atomicapp   latest              2b8075b4600e        4 hours ago         246.2 MB
```

But atomicapp still tried to download it again. Output snipped:

```
2015-08-28 18:36:33,344 - atomicapp.nulecule_base - DEBUG - Output of docker images cmd: 
2015-08-28 18:36:33,344 - atomicapp.utils - INFO - atomicapp.status.info.message=Pulling image projectatomic/mariadb-centos7-atomicapp ...
Trying to pull repository docker.io/projectatomic/mariadb-centos7-atomicapp ...
```

We need to find a way to translate the shortname into a full name and pass that full name into `docker images -q`

**Comment by vpavlin** [comment]
@rhatdan Any idea if there is a way how to get a full name of the image? (Well, other then guessing or grepping the full image list)

**Comment by goern** [comment]
@vpavlin ping

**Comment by rtnpro** [comment]
Fixed in https://github.com/projectatomic/atomicapp/tree/cdk2-beta3-rc1

**Comment by rtnpro** [comment]
@dustymabe can you close this?

**Comment by dustymabe** [comment]
Will close once code is merged to master.

**Comment by dustymabe** [comment]
d172671 was merged to master in 90a1e94
